modules/openapi-generator/src/main/resources/python/README.mustache

```diff
@@ -34,6 +34,16 @@ Then import the package:
 import {{{packageName}}}
 ```
 
+{{#detectAmznApiGatewayExtensions}}
+### SigV4
```

Comment:
arent you using withSigv4 in java? If you are using this np, just make sure to update java one as well.